### ISSUE TITLE
Remove unused `alt-hero` field from 100-automations.md project file

### ISSUE THREAD

**Issue by JessicaLucindaCheng** [hackforla/website#2891]
### Overview
As a developer, I want to make sure unused code is removed from the codebase in project cards so that
- time to load the website is as low as possible,
- it's easier to understand the code, and
- time to review the code is reduced.

### Details
In project md files, `image-hero` is a decorative background image and `alt-hero` is the alt text associated with `image-hero`. However, `alt-hero` text is not used anywhere in our codebase since the `image-hero` is used as an SCSS background image.

Moreover, if the `image-hero` were to be used in an img HTML tag in the future, the alt text for all the `image-hero` images would be `alt=""`, which could be set programmatically for all the alt properties associated with each `image-hero`.  

### Action Items
- [x] For the file `_projects/100-automations.md`, remove line 8 for the alt-hero
```
alt-hero: 'Five intermeshed gears'
```
- [x] Once the pull request associated with this issue is approved and merged, please update and edit epic #2722 by
  - [x] Checking off the dependency for this issue
  - [ ] If all dependencies are checked off, please move issue #2722 to the New Issue Approval column and remove the Dependency label

### Resources/Instructions
- This issue is part of epic #2722 

#### File and code links you will need to work on this issue
- Directory to find the page once you have it in your IDE: `_projects/100-automations.md`
- Click on the link below to see the source file code:
https://github.com/hackforla/website/blob/427c0c8ae104955f91a41e9af0fc4594cce08c22/_projects/100-automations.md?plain=1#L8

#### Information about WCAG
- [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/)
- [Web Accessibility Tutorials: Decorative Images](https://www.w3.org/WAI/tutorials/images/decorative/)
